pages/get-involved.tsx

```diff
@@ -3,20 +3,115 @@ import {
   Button,
   Container,
   Grid,
+  List,
   Typography,
 } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
 import React from 'react';
 
 import IndividualPageHead from '../components/helper/IndividualPageHead.tsx';
 import ShareButtons from '../components/helper/ShareButtons.tsx';
 import AccordionBuilder from '../components/layout/AccordionBuilder.tsx';
+import BenefitListItem from '../components/layout/BenefitListItem.tsx';
 import FactCard from '../components/layout/FactCard.tsx';
 import HeroBanner from '../components/layout/HeroBanner.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import PaperSection from '../components/layout/PaperSection.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/get-involved.ts';
 
+function BenefitsOfJoiningUs() {
+  const theme = useTheme();
+  const matchesXS = useMediaQuery(theme.breakpoints.down('sm'));
+  const matchesSM = useMediaQuery(theme.breakpoints.down('md'));
+
+  let bottomImageAdjust;
+  if (matchesSM) {
+    bottomImageAdjust = '9.8rem';
+  } else {
+    bottomImageAdjust = '6.6rem';
+  }
+  return (
+    <SectionContainer
+      id="benefits-of-joining-us"
+      maxWidth="md"
+    >
+      <Typography variant="h2">
+        {content.benefits.header}
+      </Typography>
+      <PaperSection sx={{
+        bgcolor: 'primary.dark', color: 'primary.contrastText',
+      }}
+      >
+        <Box sx={{
+          pl: 3, py: 4,
+        }}
+        >
+          {content.benefits.body.map((text) => (
+            <Typography
+              key={`benefitsText-${text}`}
+              variant="body1"
+              sx={{ mb: 4, mr: 2 }}
+            >
+              {text}
+            </Typography>
+          ))}
+          {matchesXS ? (
+            <Grid container alignItems="flex-start">
+              <Grid item xs={5}>
+                <Typography variant="body1" paragraph>
+                  {content.benefits.listHeader}
+                </Typography>
+              </Grid>
+              <Grid item xs={7}>
+                <ImageContainer
+                  src={content.benefits.mobileImgsrc}
+                  width={95}
+                  height={95}
+                  alt=""
+                  useImageDimensions
+                />
+              </Grid>
+            </Grid>
+          )
+            : (
+              <Typography variant="body1" paragraph>
+                {content.benefits.listHeader}
+              </Typography>
+            )}
```

Comment:
```suggestion
            <Grid container alignItems="flex-start">
              <Grid item xs={5}>
                <Typography variant="body1" paragraph>
                  {content.benefits.listHeader}
                </Typography>
              </Grid>
              <Grid item xs={7} sx={{ display: { xs: 'inherit', sm: 'none' } }}>
                <ImageContainer
                  src={content.benefits.mobileImgsrc}
                  width={95}
                  height={95}
                  alt="Alt text here"
                  useImageDimensions
                />
              </Grid>
            </Grid>
```
At this part we have a repetitive (`Typrography`) code that dont seems to be necessary, we can just use `sx` in Grid,  where Image is displayed, to hide this in breakpoints that we need.

Code will be more concise and less complex to read. What do you think?